code/game/machinery/cryopod.dm

```diff
@@ -302,6 +302,10 @@
 				preserve = 1
 			else
 				continue
+		else if(istype(W, /obj/item/device/pda))
```

Comment:
This issue is an issue with PDAs, not with the cryopod.

Make PDAs save the list of consoles they're attached to (via lazylist) and then, in PDA/Destroy(), go through and unset the refs. Or, actually, see about using onDestroy() even if possible. I forget if it works that way or not.